actions/workflows/st2_pkg_promote_all.yaml

```diff
@@ -111,19 +109,6 @@ tasks:
     next:
       - when: <% succeeded() %>
         publish: promoted_st2web=<% ctx().version + '-' + result().output.revision %>
-        do: promote_st2flow
-      - when: <% failed() %>
-        do: promote_st2flow
-  promote_st2flow:
```

Comment:
@m4dcoder Any specific reason behind removing promoting st2flow packages?

Reply:
@armab Because this is a duplicate. We already promote_st2flow at bwc_pkg_promote_all @ https://github.com/StackStorm/st2ci/blob/master/actions/workflows/bwc_pkg_promote_all.yaml#L120. Since st2flow is enterprise only, it does not belong in this WF for community packages.

Reply:
@m4dcoder Makes sense, thanks for the info.